### PR TITLE
DOC: Clarify definition of kurtosis used

### PR DESCRIPTION
**Pull request by bashtage**
Added explation that kurtosis returned is Fisher's kurtosis, also
known as excess kurtosis.

Also cleans up the kurtosis function to remove some unused computations.

### CODE REVIEW

**Comment by bashtage** [comment]
Trivial PR to better explain how kurtosis is computed.  When checking the calculation, I noticed 2 lines that were unused to also removes those.

File: pandas/core/generic.py [hunk 1]
```diff
@@ -3998,7 +3998,9 @@ def stat_func(self, axis=None, skipna=None, level=None,
             nanops.nanskew)
         cls.kurt = _make_stat_function(
             'kurt',
-            'Return unbiased kurtosis over requested axis\nNormalized by N-1',
+            'Return unbiased kurtosis over requested axis using Fisher''s '
+            'definition of\nkurtosis (kurtosis of normal == 0.0). Normalized '
+            'by N-1\n'',
```
Comment (shoyer) [inline pandas/core/generic.py]: this line is a syntax error

File: pandas/core/nanops.py [hunk 2]
```diff
@@ -507,11 +507,9 @@ def nankurt(values, axis=None, skipna=True):
 
     A = values.sum(axis) / count
     B = (values ** 2).sum(axis) / count - A ** 2
-    C = (values ** 3).sum(axis) / count - A ** 3 - 3 * A * B
     D = (values ** 4).sum(axis) / count - A ** 4 - 6 * B * A * A - 4 * C * A
```
Comment (shoyer) [inline pandas/core/nanops.py]: actually, `C` is used on this line

**Comment by bashtage** [comment]
Indeed on the first use of `C`.

**Comment by jreback** [comment]
@bashtage thanks

**Pull request merged**